server/utils/README.md

```diff
@@ -0,0 +1,7 @@
+### **The utils directory will serve as a location for mutltiple utilty tools.  get_request_data_csv.py can be used to make an API call to fetch 311 request data from the 311 data server for a given start date and end date.** 
+
+### The 311 request data from [lacity.org](https://data.lacity.org/browse?q=MyLA311%20Service%20Request%20Data%20&sortBy=relevance) has 34 columns. The get_request_data_csv.py script can be run from the command line passing the arguments start_date and end_date that lets you retreive the 311 request data from the [311 data server](https://dev-api.311-data.org/docs). The 311 server processes the data from lacity.org. The data cleaning procedure is mentioned [here](https://github.com/hackforla/311-data/blob/dev/docs/data_loading.md). The result is written to a csv file and saved in the current working directory of the user. A preview of the data_final dataframe is printed in the command line. 
+
+### Example: "`"python get_311_request_data_csv.py "2021-01-01" "2021-01-03"`" will return 261 rows and 15 columns.
```

Comment:
In my previous example, I said "\`" so that GitHub wouldn't automatically turn it into code. You can just use `.

Reply:
Sounds good!